front/cypress/e2e/idea_edit_page.cy.ts

```diff
@@ -25,22 +25,24 @@ describe('Idea edit page', () => {
       })
       .then((project) => {
         projectId = project.body.data.id;
-        return cy.apiCreateIdea(
-          projectId,
-          ideaTitle,
-          ideaContent,
-          undefined,
-          undefined,
-          jwt
-        );
-      })
-      .then((idea) => {
-        ideaId = idea.body.data.id;
-        ideaSlug = idea.body.data.attributes.slug;
-        cy.wait(2000);
       });
   });
 
+  beforeEach(() => {
+    cy.apiCreateIdea(
+      projectId,
+      ideaTitle,
+      ideaContent,
+      undefined,
+      undefined,
+      jwt
+    ).then((idea) => {
+      ideaId = idea.body.data.id;
+      ideaSlug = idea.body.data.attributes.slug;
+      cy.wait(2000);
+    });
+  });
+
```

Comment:
Could we add an "after each" where we delete the related idea as well? Our "after" would cover just the last created idea now right?

```
  after(() => {
    cy.apiRemoveIdea(ideaId);
  });
```

Reply:
Initially considered that deleting the project at the end would clear out all the ideas but now that you mention it, It is better to run each test with a reset environment. Pushing up a change.